src/index/drop.rs

```diff
@@ -64,14 +64,14 @@ impl IndexDropStatement {
 
 impl SchemaStatementBuilder for IndexDropStatement {
     fn build<T: SchemaBuilder>(&self, schema_builder: T) -> String {
-        let mut sql = SqlWriter::new();
+        let mut sql = String::with_capacity(256);
```

Comment:
I feel like we want to extract this as a common function, or at least extract the default string size as a constant